packages/IMAPClient-Tests.package/ICEmailTest.class/README.md

```diff
@@ -0,0 +1 @@
+This test the class ICEmail by testing basic formatting, header and body accessing. 
```

Comment:
This conducts testing of basic formatting, header and body accessing methods of the ICEmail class.